src/util/swc.rs

```diff
@@ -124,3 +126,64 @@ pub fn module_export_name_value(
 pub fn has_ignorable_js_doc_tag(js_doc: &JsDoc) -> bool {
   js_doc.tags.iter().any(|t| matches!(t, JsDocTag::Ignore) || matches!(t, JsDocTag::Unsupported { value } if value == "@internal" || value.starts_with("@internal ")))
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
+  #[test]
+  fn remove_stars_from_js_doc_works() {
+    assert_eq!(
+      remove_stars_from_js_doc(
+        "/**
+ * This module provides the `Result` class
+ */"
+      ),
+      "/**
+This module provides the `Result` class
+/"
+    );
+    assert_eq!(
+      remove_stars_from_js_doc(
+        r#"/**
+ * # Program
+ *
+ * description
+ *
+ * ## Usage
+ *
+ * @example
+ * ```ts
+ * import * as mod from "program";
+ */"#
+      ),
+      r#"/**
+# Program
+
+description
+
+## Usage
+
+@example
+```ts
+import * as mod from "program";
+/"#
+    );
+    assert_eq!(
+      remove_stars_from_js_doc(
+        r#"/**
+ * # Program
+ * **Example:**
+ *
+ * example1
+ */
+"#
+      ),
+      r#"/**
+# Program
+**Example:**
```

Comment:
this doesnt seem to be getting stripped, unlike what you mentioned in your previous comment

Reply:
Yes what I had is 
```
/**
** Example** // no starting star 
*/
```

But then I guess maybe we can consider this a user error to not use the star at the start